.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+**Version Used**: 
+
+**Summary:**
+
+
+**Steps to Reproduce**:
+
+1. 
+
+2. 
+
+3. 
+
+**Expected Behavior**:
+
+**Actual Behavior**:
+
+**User Impact**:
```

Comment:
Most of time the issues only say something like "Variable X is set to Y but it should Z". The hope is that by explicitly calling out user impact people will be reminded to write it down like "Customers are getting the wrong error because variable X is set to Y"

Reply:
Is this issue template for us or for externals?

Reply:
I think it is for everyone but i suspect we will use it the most